### DISCUSSION TITLE
Wait! Double check your Remote URL [http://localhost/D:\UnityDemo\hot update\PATCHWorkspace\Builds\0.1.2\Game] before proceeding. It's not reachable by the Launcher! The specific file the Launcher is trying to access is: [http://localhost/D:\UnityDemo\hot update\PATCHWorkspace\Builds\0.1.2\Game/Builds/builds_index.json] UnityEngine.Debug:LogError (object) MHLab.Patch.Admin.Editor.Inspectors.LauncherDataCustomInspector:CheckRemoteUrl (MHLab.Patch.Core.Client.ILauncherSettings) (at Assets/MHLab/Patch/Admin/Editor/Inspectors/LauncherDataCustomInspector.cs:42) MHLab.Patch.Admin.Editor.Inspectors.LauncherDataCustomInspector:OnInspectorGUI () (at Assets/MHLab/Patch/Admin/Editor/Inspectors/LauncherDataCustomInspector.cs:22) UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

### DISCUSSION THREAD

**Discussion by 2651362524** [emanzione/PATCH#73] [General]
Wait! Double check your Remote URL [http://localhost/D:\UnityDemo\hot update\PATCHWorkspace\Builds\0.1.2\Game] before proceeding. It's not reachable by the Launcher! The specific file the Launcher is trying to access is: [http://localhost/D:\UnityDemo\hot update\PATCHWorkspace\Builds\0.1.2\Game/Builds/builds_index.json]
UnityEngine.Debug:LogError (object)
MHLab.Patch.Admin.Editor.Inspectors.LauncherDataCustomInspector:CheckRemoteUrl (MHLab.Patch.Core.Client.ILauncherSettings) (at Assets/MHLab/Patch/Admin/Editor/Inspectors/LauncherDataCustomInspector.cs:42)
MHLab.Patch.Admin.Editor.Inspectors.LauncherDataCustomInspector:OnInspectorGUI () (at Assets/MHLab/Patch/Admin/Editor/Inspectors/LauncherDataCustomInspector.cs:22)
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

**Comment by emanzione** [comment]
Hello.
Without further details, context and/or explanation the only thing I could say is that your Remote URL is wrong.
